VHbbAnalysis/Heppy/python/VHbbAnalyzer.py

```diff
@@ -443,10 +447,10 @@ def process(self, event):
               j.btagIdx=-1
       
 	#substructure threshold, make configurable
-	ssTrheshold = 200.
+	ssThreshold = 200.
 	# filter events with less than 2 jets with pt 20
         event.jetsForHiggs = [x for x in event.cleanJets if self.cfg_ana.higgsJetsPreSelection(x) ]
-	if not   len(event.jetsForHiggs) >= 2 : # and event.jetsForHiggs[1] > 20.) : # or(len(event.cleanJets) == 1 and event.cleanJets[0] > ssThreshold ) ) :
+	if not  ( len(event.jetsForHiggs) >= 2  or (len(event.cleanJets) == 1 and event.cleanJets[0] > ssThreshold ) ) :
```

Comment:
isn't .pt() missing here, or is there some heppy magic I dont know about?  i.e. event.cleanJets[0]**.pt()** > ssThreshold